truss/templates/shared/dynamic_config_resolver.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+from typing import Optional
+
+DYNAMIC_CONFIG_MOUNT_DIR = "/etc/dynamic_config"
```

Comment:
nit: wondering if we should prefix with `b10_` to ensure it won't collide with anything else
```suggestion
DYNAMIC_CONFIG_MOUNT_DIR = "/etc/b10_dynamic_config"
```